maintain/going-production-with-k8s.md

```diff
@@ -0,0 +1,429 @@
+  volumes with replication factor 2 or replication factor 3
+  
+  Here is how one can configure a volume with replication factor 3 for e.g.,
+  
+  ```
+  sudo /opt/pwx/bin/pxctl volume create dbasevol --size=1 --repl=3 --iopriority=high
+  ```
+  
+* PX makes best effort to distribute volumes evenly across all nodes and based on the `iopriority` that is requested. When
+  PX cannot find the appropriate media type that is requested to create a given `iopriority` type, it will attempt to
+  create the volume with the next available `iopriority` level. 
+  
+* Volumes can be created in different availability zones by using the `--zones` option in the `pxctl volume create` command
+
+  ```
+  sudo /opt/pwx/bin/pxctl volume create dbasevol --size=1 --repl=3 --iopriority=high --zones=us-east-1,us-east-2,us-east-3 
+  ```
+* Volumes can be created in different racks using `--racks` option and passing the rack labels when creating the volume
+
+  ```
+  sudo /opt/pwx/bin/pxctl volume create dbasevol --size=1 --repl=3 --iopriority=high --racks=dcrack1,dcrack2,dcrack3 
+  ``` 
+  Please ensure the PX containers are started with the corresponding rack parameters.
+ 
+* If the volumes need to be protected against accidental deletes because of background garbage collecting scripts, 
+  then the volumes need to enabled with `--sticky` flag
+  
+  ```
+   sudo /opt/pwx/bin/pxctl volume create dbasevol --size=1 --repl=3 --iopriority=high --sticky
+  ```
+  
+* The `--sticky` flag can be turned on and off using the `pxctl volume update` commands
+ 
+  ```
+  sudo /opt/pwx/bin/pxctl volume update dbasevol --sticky=off
+  ```
+  
+* For applications that require shared access from multiple containers running in different hosts, 
+  Portworx recommends running  shared volumes. Shared volumes can be configured as follows:
+
+  ```
+  pxctl volume create wordpressvol --shared --size=100 --repl=3
+  ```
+  This [page](https://docs.portworx.com/manage/volumes.html) gives more details on different volume types, 
+  how to create them and update the configuration for the volumes 
+
+### Data Protection for Containers
+
+* Snapshots - Follow DR best practices and ensure volume snapshots are scheduled for instantaneous recovery in the 
+  case of app failures. 
+  
+* Portworx support 64 snapshots per volume
+
+* Each snapshot can be taken manually via the `pxctl snap create` command. For e.g.,
+
+  ```
+  pxctl snap create --name mysnap --label color=blue,fabric=wool myvol
+  Volume successfully snapped: 1152602487227170184
+  ```  
+* Alternatively, snapshots can be scheduled by creating a hourly, daily or weekly schedule. This will enable the snapshots 
+  to be automatically created without user intervention.
+  
+  ```
+  pxctl volume create --daily @08:00 --daily @18:00 --weekly Friday@23:30 --monthly 1@06:00 myvol
+  ``` 
+* Here is more information on how to setup [snapshots](https://docs.portworx.com/manage/snapshots.html) in PX-Enterprise.
```

Comment:
Duplication of info. We can just point to the existing page: https://docs.portworx.com/scheduler/kubernetes/snaps.html